File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/GenerationStyleUpdateForm.java
package com.sdkwork.app.model;

public class GenerationStyleUpdateForm {
    private String name;
    private String description;
    private Map<String, Object> configParams;

    public String getName() {
        return this.name;
    }
    
    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return this.description;
    }
    
    public void setDescription(String description) {
        this.description = description;
    }

    public Map<String, Object> getConfigParams() {
        return this.configParams;
    }
    
    public void setConfigParams(Map<String, Object> configParams) {
        this.configParams = configParams;
    }
}

File: sdkwork-app-sdk-csharp/Models/GenerationStyleUpdateForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class GenerationStyleUpdateForm
    {
        public string? Name { get; set; }
        public string? Description { get; set; }
        public Dictionary<string, object>? ConfigParams { get; set; }
    }
}
